### ISSUE TITLE
statistical test for p-value 

### ISSUE THREAD

**Issue by agehrke6** [GreenleafLab/chromVAR#46]
Hi Alicia,

Could you please let me know the name of the statistical test performed in the computeVariability function?  Am I correct that it is a Brown-Forsythe test?

Thanks for your help,

-Andrew

**Comment by AliciaSchep** [comment]
Yes, if you are referring to `differentialVariability` that is! That would be a good thing to add to the docs.
